src/constants/grid.ts

```diff
@@ -13,16 +13,16 @@ const grid: IGrid = {
     l: 992,
     m: 768,
     s: 576,
-    xl: 1200,
+    xl: 1250,
```

Comment:
the xl sizes and the gutter size look a bit weird to me :|

Reply:
Might be worth speaking to the designers about this.
This update is based on the most recent designs for shopfront.

Reply:
turns out the gutter spacing is 24 not 22 so I have updated the breakpoint to 1300 which is slightly less of a random number :D